src/uncrustify_types.h

```diff
@@ -263,7 +263,7 @@ struct chunk_t
    }
 
 
-   int len()
+   size_t len()
```

Comment:
This and the return type of size() in unc_text.h should be deque<int>::size_type.